index.html

```diff
@@ -78,6 +78,102 @@ <h3>HTML 4.0 (1997-1999)</h3>
         </ul>
       </section>
       <hr />
+      <section>
+        <h2>The Rise of Modern HTML</h2>
+        <h3>XHTML (2000)</h3>
+        <p>
+          A stricter version of HTML based on <strong>XML</strong>, designed for
+          cleaner coding. However, it was considered too rigid and never gained
+          wide adoption.
+        </p>
+        <h3>HTML5 (2014 - Present)</h3>
+        <p>The most significant update to HTML, bringing:</p>
+        <p><strong>Key Features of HTML5:</strong></p>
+        <ol>
+          <li><strong> Native multimedia</strong> support</li>
+          <li><strong> Semantic elements</strong></li>
+          <li><strong> Canvas and SVG</strong> for graphics and animations</li>
+          <li><strong> Improved forms</strong></li>
```

Comment:
You got some extra spaces here:
```suggestion
          <li><strong>Native multimedia</strong> support</li>
          <li><strong>Semantic elements</strong></li>
          <li><strong>Canvas and SVG</strong> for graphics and animations</li>
          <li><strong>Improved forms</strong></li>
```

Reply:
I wanted this ordered list to more closely resemble the Google Doc - that is why I added the extra spaces.